data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/OpenSearchSinkIT.java

```diff
@@ -399,39 +399,45 @@ private Integer getDocumentCount(final String index, final String field, final S
     return mappings;
   }
 
+  @SuppressWarnings("unchecked")
   private String getIndexPolicyId(final String index) throws IOException {
+    // TODO: replace with new _opensearch API
     final Request request = new Request(HttpMethod.GET, "/_opendistro/_ism/explain/" + index);
     final Response response = client().performRequest(request);
     final String responseBody = EntityUtils.toString(response.getEntity());
 
-    @SuppressWarnings("unchecked")
-    final String policyId = (String) ((Map<String, Object>)createParser(XContentType.JSON.xContent(),
-            responseBody).map().get(index)).get("index.opendistro.index_state_management.policy_id");
+    String policyId = (String) ((Map<String, Object>)createParser(XContentType.JSON.xContent(),
+            responseBody).map().get(index)).get("index.plugins.index_state_management.policy_id");
+    // If backend is ODFE
+    if (policyId == null) {
+      policyId = (String) ((Map<String, Object>)createParser(XContentType.JSON.xContent(),
+              responseBody).map().get(index)).get("index.opendistro.index_state_management.policy_id");
+    }
     return policyId;
   }
 
-  public static boolean isODFE() {
-    final boolean odfeFlag = Optional.ofNullable(System.getProperty("odfe"))
+  public static boolean isOSBundle() {
+    final boolean osFlag = Optional.ofNullable(System.getProperty("os"))
```

Comment:
Multiple references to `os` - might be better to not abbreviate if possible to avoid overloaded abbreviation (e.g. "operating system").